test/azure/legacy/Expected/AcceptanceTests/PackageModeDataPlane/README.md

```diff
@@ -0,0 +1,53 @@
+
+# Azure Package Mode client library for Python
+<!-- write necessary description of service -->
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please 
+refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install packagemode
+```
+
+#### Prequisites
+
+- Python 3.6 or later is required to use this package.
```

Comment:
PackageModeDataPlane should be in `vanilla`, not `azure`

Reply:
Will do it. I am not afraid that I don't quite understand why need to put it in vanilla.

Reply:
I agree with @iscai-msft , though the lines have been blurried more recently. For isntance, we support LRO in data-plane, while for autorest LRO is "azure" (mgmt) feature.